central/metrics/custom/tracker/tracker_base.go

```diff
@@ -269,10 +274,34 @@ func (tracker *TrackerBase[Finding]) Gather(ctx context.Context) {
 	if cfg.period == 0 || time.Since(gatherer.lastGather) < cfg.period {
 		return
 	}
+	begin := time.Now()
 	if err := tracker.track(ctx, gatherer.registry, cfg.metrics); err != nil {
 		log.Errorf("Failed to gather %s metrics: %v", tracker.description, err)
 	}
-	gatherer.lastGather = time.Now()
+	end := time.Now()
+	gatherer.lastGather = end
+
+	titCat := strings.ToTitle(tracker.description[0:1]) + tracker.description[1:]
```

Comment:
I feel like there must be a better name than `titCat` for this variable. Maybe just write it out `titleCategory`?